build.redis.xml

```diff
@@ -30,4 +33,10 @@
             </else>
         </if>
     </target>
+
+    <target name="redis-flush">
+        <echo msg="Cleaning redis flush." />
```

Comment:
Not so sure about the phrasing. Aren't "clean" and "flush", as verbs, the same in this context? Also "Redis" is a name and should be "word uppercased". Maybe "Flushing Redis backend"?